tests/test_integration_crashpad.py

```diff
@@ -2,19 +2,14 @@
 import os
 import time
 from . import make_dsn, run, Envelope
-from .conditions import has_crashpad, has_http
+from .conditions import has_crashpad
 from .assertions import assert_session
 
 # TODO:
-# * with crashpad backend:
-#   - breadcrumbs, attachments, etc
-#   - crash
-#   - expect report via http
+# Actually assert that we get a correct event/breadcrumbs payload
 
 
-@pytest.mark.skipif(
-    not has_http or not has_crashpad, reason="test needs crashpad backend"
-)
+@pytest.mark.skipif(not has_crashpad, reason="test needs crashpad backend")
```

Comment:
FYI, I think you only use this twice but if you need it more you can:
```
needs_crashpad = pytest.mark.skipif(not has_crashpad, reason="test needs crashpad backend")

@needs_crashpad
def test_crashpad_capture(...):
    ....
```

Reply:
Oh, that is nice. We do have quite a few repetitive `skipif` in the tests.